csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests/runtest.sh

```diff
@@ -45,23 +48,23 @@ if [ $RunTestNative = "true" ]; then
 
   inc="-I build/native/include"
 
-  if [ $IsMacOS = "true" ]; then
+  if [[ $IsMacOS == "True" || $IsMacOS == "true" ]]; then
```

Comment:
looks like there is a way to convert to lower case: https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html#Shell-Parameter-Expansion (search for ${parameter,,pattern})
how about `${IsMacOs,,} == "true"`?